src/olympia/editors/tests/test_views.py

```diff
@@ -1512,12 +1522,10 @@ def test_preserve_multi_apps(self):
                               amo.STATUS_NOMINATED, amo.STATUS_AWAITING_REVIEW,
                               application=app, listed=self.listed)
 
-        r = self.search(application_id=[amo.MOBILE.id])
```

Comment:
This test was obsolete (it's looking for content in a column that was removed in #3831), but we didn't notice because it was skipped before.